src/lib/ReactDataTable/ReactDataTable.tsx

```diff
@@ -40,167 +46,223 @@ const ReactDataTable = <TData,>(props: ReactDataTableProps<TData>) => {
     }
   }`;
 
+  const DraggableRow = ({ row, table }: { row: Row<TData>; table: Table<TData> }) => {
+    if (!table.options.getRowId) {
+      throw new Error("You must provide 'getRowId()' to data-table options in order to use the drag-and-drop feature.");
+    }
+
+    const { transform, transition, setNodeRef, isDragging } = useSortable({
+      id: row.id,
+    });
+
+    const draggableStyle: CSSProperties = {
+      transform: CSS.Transform.toString(transform),
+      transition: transition,
+      opacity: isDragging ? 0.8 : 1,
+      zIndex: isDragging ? 1 : 0,
+      position: "relative",
+    };
+
+    return (
+      <tr key={row.id} ref={setNodeRef} style={rowStyle ? { ...rowStyle(row.original), ...draggableStyle } : draggableStyle}>
+        {row.getVisibleCells().map((cell) => (
```

Comment:
maybe move this part to a small function? i just added `cellClassName` in another pr, and else its easy possible to forget having to add it at multiple places.

Reply:
Thanks, done ✓